packages/identity/src/identity.js

```diff
@@ -307,9 +307,17 @@ export async function getPublicIdentity(tankerIdentity: b64string): Promise<b64s
   throw new InvalidArgument(`Invalid secret identity provided: ${tankerIdentity}`);
 }
 
-export function upgradeIdentity(tankerIdentity: b64string): b64string {
+export async function upgradeIdentity(tankerIdentity: b64string): b64string {
   if (!tankerIdentity || typeof tankerIdentity !== 'string')
     throw new InvalidArgument('tankerIdentity', 'b64string', tankerIdentity);
-  const identity = _deserializeIdentity(tankerIdentity);
+  const frozenIdentity = _deserializeIdentity(tankerIdentity);
+  // $FlowIgnore flow doesn't understand that the spread doesn't change the type
```

Comment:
We don't use flow anymore, do we?

Reply:
On master, I still see the following in `package.json`:

`"proof": "yarn lint:js && yarn lint:flow && yarn test"`

I'm not aware of the status of the Typescript migration, but at least as of this feature branch we still need Flow to pass the CI lint, I believe

Reply:
the typescript pr https://github.com/TankerHQ/identity-js/pull/6 is not merged yet. It stills need a little testing (ie. target -> browser version). I'll handle the conflicts once this is merged